releasenotes/notes/beta-to-stable-bcae689e8700c667.yaml

```diff
@@ -0,0 +1,236 @@
+    to be made to the ansible inventory or group_vars to support stable cert-manager
+
+    .. code-block:: yaml
+
+      cert_manager_controller_extra_args:
+        - "--enable-gateway-api"
+
+    In previous builds the ``--enable-gateway-api`` was unset, but it is now a
+    required option.
+  - |
+    When upgrading from a pre-release to stable, the following changes will need
+    to be made to the ansible inventory or group_vars to support stable metallb
+
+    .. code-block:: yaml
+
+      metallb_enabled: false
+
+    In previous builds the ``metallb_enabled`` was set to true, but it is now
+    managed by the MetalLB helm chart.
+  - |
+    When upgrading from a pre-release to stable, the following changes will need
+    to be made to the ansible inventory or group_vars to eliminate the CNI plugin
+    from the Kubespray Management.
+
+    .. code-block:: yaml
+
+      kube_network_plugin: none
+
+    In previous builds the ``kube_network_plugin`` was set to kube-ovn, but it is
+    now managed by the Kube-OVN helm chart.
+  - |
+    When upgrading from a pre-release to stable, the following changes will need
+    to be made to the ansible inventory or group_vars to eliminate the previous
+    assumption of the kubeadm patch files.
+
+    .. code-block:: yaml
+
+      kubeadm_patches: []
+
+    In previous builds the ``kubeadm_patches`` was set to a dictionary of patches
+    that would deploy files into the environment. This interface was changed
+    upstream and now must be a list of string type patches. Review the upstream
+    documentation[0] for more information.
+
+    [0]: https://github.com/kubernetes-sigs/kubespray/blob/v2.27.0/roles/kubernetes/kubeadm_common/defaults/main.yml
+  - |
+    When upgrading from a pre-release to stable, the following file no longer has
+    any effect on the environment and can be removed from the ansible group_vars.
+
+    * /etc/genestack/inventory/group_vars/k8s_cluster/k8s-net-kube-ovn.yml
+
+    This file can be eliminated.
+  - |
+    When upgrading from a pre-release to stable, review the Kube-OVN to Helm
+    migration documentation at https://docs.rackspacecloud.com/k8s-cni-kube-ovn-helm-conversion
+    as this step will be required before running Kubespray again.
+deprecations:
+  - |
+    In early builds of Genestack Kube-OVN was deployed and managed by Kubespray;
+    however, this is no longer the case. The Kube-OVN helm chart is now used to
+    deploy and manage the Kube-OVN CNI plugin.
+  - |
+    In early builds of Genestack MetalLB was deployed and managed by Kubespray;
+    however, this is no longer the case. The MetalLB helm chart is now used to
+    deploy and manage the MetalLB LoadBalancer.
+  - |
+    In early builds of Genestack the cert-manager option ``ExperimentalGatewayAPISupport``
+    was set to true, within the ansible group_vars. This option should be removed as it
+    no longer has any effect.
```

Comment:
Do we want to mention 'known issues' like OVN loadbalancers configurations unstable? Or possibly some of the shortcomings in skyline, like missing octavia features.

Reply:
lets do it

Reply:
done.